react/components/MadieAlert/index.jsx

```diff
@@ -152,9 +186,11 @@ MadieAlert.propTypes = {
     alertProps: PropTypes.object,
     closeButtonProps: PropTypes.object,
     copyButton: PropTypes.bool,
+    MinimizeAlerts: PropTypes.bool,
 };
 MadieAlert.defaultProps = {
     copyButton: false,
+    MinimizeAlerts: false,
```

Comment:
for consistency can we use same casing
```suggestion
    minimizeAlerts: false,
```